src/Bicep.Core/Registry/OciModuleRegistry.cs

```diff
@@ -106,6 +107,12 @@ public override bool TryGetLocalModuleEntryPointUri(OciArtifactModuleReference r
             if (!ociAnnotations.Any() || (ociAnnotations.TryGetValue(LanguageConstants.OciOpenContainerImageDocumentationAnnotation, out string? documentationUri) &&
                 string.IsNullOrWhiteSpace(documentationUri)))
             {
+                if (ociArtifactModuleReference.Registry == LanguageConstants.McrRegistry && ociArtifactModuleReference.Repository.StartsWith(LanguageConstants.McrRepositoryPrefix))
```

Comment:
>StartsWith [](http://example.com/codeflow?start=130&length=10)

Can you pass in the Ordinal comparison/comparer here to match `OciArtifactModuleReference`?

Reply:
Sure, updated here:
https://github.com/Azure/bicep/pull/9709/commits/77c5456003e7e23edfb78eac7227ce74674597c7

Thanks!